### PR TITLE
Add email about COVID-19

### PR DESCRIPTION
**Pull request by RealOrangeOne**
Related to https://github.com/srobo/team-emails/pull/47

### CODE REVIEW

File: SR2020/2020-03-13-covid-19.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+- If we can't get enough teams to come
+- If we can't get enough volunteers to come
+- We don't think we can offer a safe event
+
+
+We expect all those attending the competition to follow the [guidelines][phe-guidelines] on COVID-19 from Public Health England.
+
+With that in mind, we'd like to confirm the opinions of our volunteers, to ensure we make the correct, most informed decision.
+
+Could you fill out the below, 2 question survey around your stance on volunteering for SR2020: https://forms.gle/TjW2KRzgH6xBKv1v6
```
Comment (antoinepetty) [inline SR2020/2020-03-13-covid-19.md]: Is there a way to make this stand out more in the wall of text? Needs to be clear that this is important to be actioned

File: SR2020/2020-03-13-covid-19.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+- If the venue cancels our booking
+- If we can't get enough teams to come
+- If we can't get enough volunteers to come
+- We don't think we can offer a safe event
+
+We expect all those attending the competition to follow the [guidelines][phe-guidelines] on COVID-19 from Public Health England.
+
+With that in mind, we'd like to confirm the opinions of our volunteers, to ensure we make the correct, most informed decision.
+
+Could you fill out the below, 2 question survey around your stance on volunteering for SR2020:
```
Comment (antoinepetty) [inline SR2020/2020-03-13-covid-19.md]: Thanks for the button - should this text be bold as well or is that too much?

Comment (RealOrangeOne) [reply SR2020/2020-03-13-covid-19.md]: I think that might be too much. The button is probably eye-catching enough.

Comment (antoinepetty) [inline SR2020/2020-03-13-covid-19.md]: Very minor: maybe add "please" like on the teams email

File: SR2020/2020-03-13-covid-19.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+Hello everyone,
+
+I'm sure you've heard about the COVID-19 coronavirus, and about many large events being cancelled due to concerns of spreading the virus.
+
+We're keeping a keen eye on the situation, and will be following the advice of the [UK Department of Health and Social Care][uk-coronavirus-page]. Considering this is an event for minors, we will be taking decisions with an abundance of caution.
+
+We have put together a page on our website expanding further on how we're assessing the situation. You can find it here: https://studentrobotics.org/covid-19/
+
+We will be keeping this page up to date as things progress. As it stands the competition is going ahead as planned, however should this change, we will let you know immediately. The only things that will stop us are:
```
Comment (PeterJCLaw) [inline SR2020/2020-03-13-covid-19.md]: s/immediately/as soon as we can/
```suggestion
We will be keeping this page up to date as things progress. As it stands the competition is going ahead as planned, however should this change, we will let you know as soon as we can. The only things that will stop us are:
```

File: SR2020/2020-03-13-covid-19.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+- If the government says we can't/shouldn't proceed
+- If the venue cancels our booking
+- If we can't get enough teams to come
+- If we can't get enough volunteers to come
+- We don't think we can offer a safe event
+
+We expect all those attending the competition to follow the [guidelines][phe-guidelines] on COVID-19 from Public Health England.
+
+With that in mind, we'd like to confirm the opinions of our volunteers, to ensure we make the correct, most informed decision.
```
Comment (PeterJCLaw) [inline SR2020/2020-03-13-covid-19.md]: What is "correct" here? "Best" might be a better word?

File: SR2020/2020-03-13-covid-19.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+Could you please fill out the below, 2 question survey around your stance on volunteering for SR2020:
+
+[COVID-19 Poll](https://forms.gle/TjW2KRzgH6xBKv1v6)
```
Comment (PeterJCLaw) [inline SR2020/2020-03-13-covid-19.md]: Should this poll capture who the person is? Or is it intentional not to capture that?
(I don't have a strong opinion either way, just checking that it's been thought about and is intentional)

File: SR2020/2020-03-13-covid-19.md (hunk 3 above)
Comment (PeterJCLaw) [inline SR2020/2020-03-13-covid-19.md]: I keep reading "The only things that will stop us" almost like it's a challenge being issued. Maybe something like "The things would prevent the competition from running"?

File: SR2020/2020-03-13-covid-19.md (hunk 5 above)
Comment (RealOrangeOne) [reply SR2020/2020-03-13-covid-19.md]: Intentionally doesn't, just because we don't actually need that information.

**Pull request merged**